### PR TITLE
Added the cleaned version of our Swagger_UI

### PR DESCRIPTION
**Pull request by tuz666**
Nearly finished the whole documentation of Arcsi's API calls.
It seems that everything is working now.
I find some properties in the request post bodies which I thought not necessary.
I flagged them with a "fölösleges" tag, please correct me if I'm wrong.

### CODE REVIEW

File: arcsi/__init__.py [hunk 1]
```diff
@@ -64,4 +65,16 @@ def create_app(config_file):
     app.register_blueprint(api.arcsi)
     app.register_blueprint(view.router)
 
+    ### swagger specific ###
+    SWAGGER_URL = '/docs'
```
Comment (pvj) [inline arcsi/__init__.py:69]: I know this is a nitpick but can you please modify it from `docs` to `doc` ? We try to use an API endpoint naming schema where everything is singular

File: arcsi/static/docs.json [hunk 2]
```diff
@@ -0,0 +1,960 @@
+{
+    "openapi": "3.0.0",
+    "info": {
+      "description": "Arcsi API doc",
+      "version": "0.9.0",
```
Comment (pvj) [inline arcsi/static/docs.json:5]: 😸

File: arcsi/static/docs.json [hunk 3]
```diff
@@ -0,0 +1,960 @@
...
+              "type": "boolean"
+            },
+            "uploader": {
+              "type": "string"
+            },
+            "shows": {
+              "type": "integer",
+              "format": "int32"
+            },
+            "show_name": {
```
Comment (pvj) [inline arcsi/static/docs.json:946]: Where did you find this? I think you are right we can remove this

File: arcsi/static/docs.json [hunk 4]
```diff
@@ -0,0 +1,960 @@
...
+              "type": "boolean"
+            },
+            "archive_mixcloud": {
+              "type": "boolean"
+            },
+            "users": {
+              "type": "integer",
+              "format": "int32"
+            },
+            "user_name": {
```
Comment (pvj) [inline arcsi/static/docs.json:802]: These are actually POST'ed to our API from the form, maybe put something of an example to these two? 

F.e.: `user_name` might be `Arcsi` and `user_email` `arcsi[at]lahmacun[dot]hu` ?

**Review by pvj — CHANGES_REQUESTED** [review]
Hey tuz! Sorry for keeping you waiting this long.

Very very good job, wouldn't know if this is your 1st or 1000th pull request! I have 2 minor requests what do you think of those? Cheers

File: arcsi/static/docs.json (hunk 3 above)
Comment (tuz666) [reply arcsi/static/docs.json:946]: I found them in the POST methods of the Show and Item DetailsSchemas
(arcsi/api/show.py line: 146-154, 229-237; arcsi/api/item.py line: 107-110, 290-293)
I tried to remove them but as I saw without them the methods won't work correctly.

File: arcsi/static/docs.json (hunk 2 above)
Comment (tuz666) [reply arcsi/static/docs.json:5]: What's so funny here? 😨

File: arcsi/__init__.py (hunk 1 above)
Comment (tuz666) [reply arcsi/__init__.py]: Fixed it 👍

File: arcsi/static/docs.json (hunk 4 above)
Comment (tuz666) [reply arcsi/static/docs.json]: I added every kind of examples to the PostRequestBodies

**Review by pvj — CHANGES_REQUESTED** [review]
We did personal review of most of these and discussed how to continue please apply those as discussed. Thanks, great work!

**Review by pvj — APPROVED** [review]
@tuz666 You're really good librarian thanks for all the work on this! Mad!!

**Pull request merged**